.gitignore

```diff
@@ -7,4 +7,5 @@ prebuilds/
 lib/
 libexec/
 share/
+include/
```

Comment:
@piranna The SemaphoreCI build environment has a bit more space than that. :)
Let me know if I can help you out CI wise.